mlflow/__init__.py

```diff
@@ -247,3 +249,15 @@
     from mlflow import gateway  # noqa: F401
 
     __all__.append("gateway")
+
+
+def _configure_mlflow_loggers():
```

Comment:
Also isn't this redundant with the utility function that you made?

Reply:
Thanks @Flametaa , Removed Redundant code